src/main/java/oss/fosslight/service/T2UserService.java

```diff
@@ -61,5 +61,6 @@ public interface T2UserService extends UserDetailsService {
 	
 	public boolean existUserIdOrEmail(String userId);
 	public int updateUserNameDivision(T2Users userInfo);
-	public String[] checkUserInfo(T2Users userInfo) ;
+	public String[] checkUserInfo(T2Users userInfo);
+	public boolean isAdmin(String _token);
 }
```

Comment:
@brorica In the POSIX standard, a line is defined as: `A sequence of zero or more non- <newline> characters plus a terminating <newline> character`. Could you please add a newline at the end of the file to follow this standard?